### PR TITLE
Including NCO changes for consideration to be incorporated into

### PR DESCRIPTION
**Pull request by ShelleyMelchior-NOAA**
Incorporating the differences in NCO's $PACKAGEROOT/obsproc.v1.0.2 into release/obsproc.v1.0.0.

I proposed we adopt these changes now. However, the next release after WCOSS2 go-live can better address the tcvitals situation. I will create a github issue that addresses these changes better.

### CODE REVIEW

File: versions/run.ver
Comment (ilianagenkova) [inline versions/run.ver:9]: @ShelleyMelchior-NOAA , if we take NCO's versions we would have to update our trigger scripts to the same versions, or we need to rename our release branches to match  NCO's version. Comments?

**Review by ilianagenkova — CHANGES_REQUESTED** [review]
Will approve once we decide on versions/run.ver file.

File: versions/run.ver
Comment (ShelleyMelchior-NOAA) [reply versions/run.ver:9]: Correct. We need to take their numbers. I have never renamed a branch b/c it seemed kind of hairy for tracking purposes, especially if it is a branch many are working from or contributing to. I did some googling, and it is possible. I just don't know if it is worth it w/ just a little bit of time left to be working from the release branch before it is merged into develop on 6/28. We can update `run.ver` to match NCO's versions w/o renaming the release branch. The name of the release branch is moot, imo.

As for the trigger scripts, they don't need to be changed at this point. They pick up the versions for bufr_dump and prepobs from `run.ver`, which will correctly reflect NCO's numbers. The only thing that will need to be hardcoded in triggers is `export obsproc_ver=` in order to find HOMEobsproc when that var is pointing to production/NCO. In lieu or renaming any release branches, I think the trigger modifications need to be postponed post go-live. In the same breath, the setting for `export obsproc_ver=` will frequently be changing in all triggers as the obsproc software increments after implementations. I don't find it practical for all triggers to always be udpated after each increment. Instead, it should be incumbent upon all developers to understand that that var needs to be carefully considered by the trigger user to be consistent with the operational version for obsproc code. For example, an improvement to the triggers could be:
`export obsproc_ver=1.0.0            # check to set to latest operational installation in $PACKAGEROOT`

Comment (ilianagenkova) [reply versions/run.ver:9]: I was only looking for confirmation, but a solution along with it is much appreciated!

**Review by ilianagenkova — APPROVED** [review]

**Review by ilianagenkova — APPROVED** [review]

**Pull request merged**
